aiohttp/web_fileresponse.py

```diff
@@ -48,6 +48,19 @@
     {ext: mimetypes.encodings_map[ext] for ext in (".br", ".gz")}
 )
 
+FALLBACK_CONTENT_TYPE = "application/octet-stream"
+
+# https://en.wikipedia.org/wiki/List_of_archive_formats#Compression_only
+ENCODING_CONTENT_TYPES = MappingProxyType(
+    {
+        "gzip": "application/gzip",
+        "br": "application/x-brotli",
+        "bzip2": "application/x-bzip2",
+        "compress": "application/x-compress",
+        "xz": "application/x-xz",
+    }
+)
+
```

Comment:
Maybe it makes sense to let `mimetypes` handle this all

We could instead create an aiohttp specific [`mimetypes.MimeTypes`](https://docs.python.org/3/library/mimetypes.html#mimetypes.MimeTypes) object, clear the [`encodings_map`](https://github.com/python/cpython/blob/0ec761a96a80f4b84b98aad5e784e4ac8f385c3f/Lib/mimetypes.py#L417C5-L417C18) and add these types with `add_type` ex: https://github.com/python/cpython/blob/0ec761a96a80f4b84b98aad5e784e4ac8f385c3f/Lib/mimetypes.py#L80

Than you have a `mimetypes_obj.guess_type(str(self._path))` which always returns the `ct` expected below

Reply:
Good idea, but might as well just call the methods directly.  No need to complicate with a subclass. 👍🏻